Microsoft.Toolkit.Uwp.UI.Controls.Graph/Microsoft.Toolkit.Uwp.UI.Controls.Graph.csproj

```diff
@@ -13,13 +13,19 @@
     <None Remove="Assets\genericfile.png" />
     <None Remove="Assets\person.png" />
     <None Remove="Assets\photo.png" />
+    <None Remove="PlannerTaskList\PlannerTaskList.xaml" />
+    <None Remove="PowerBIEmbedded\EmbedReportFrame.html" />
+    <None Remove="PowerBIEmbedded\jquery-3.2.1.min.js" />
```

Comment:
This looks like a csproj artifact since we removed the dependency?

Reply:
@michael-hawker jquery was removed, but that appears these items weren't removed automatically from the .csproj file, but since it's marked as "Remove" so it doesn't impact the project too, anyway, I just removed it from the .csproj file.